index.js

```diff
@@ -37,19 +38,32 @@ module.exports.iMeanMost =
 module.exports.butWhatActuallyIsCinnamonToastCrunchThough =
   function nobodyEvahExplainedItToMe() {
     return "Scientists still aren't sure, but we have a suspicion that it " +
-      "just might be a cereal involving cinnamon, toast, and something " +
-      "crunchy!"
+      "just might be something edible involving cinnamon, toast, and something " +
+      "umm... crunchy!"
   }
 
 module.exports.butWhyIsCinnamonToastCrunchGood = 
   function nobodyEvahExplainedThisToMeEither () {
-    return "It's simply a fact of life, you know? Birds fly, gravity falls, " +
-      "stevens universe, Cinnamon Toast Crunch is good."
+    return "It's simply a fact of life, you know? Birds fly, dogs walk, " +
```

Comment:
@towerofnix I wouldn't jump from topic to topic. I feel it's more fun if we just talk about animals then suddenly jump to Cinnamon Toast Crunch. I don't know what you think..

Reply:
This has nothing to do with milk!! We also can't just change which universal truths we take to be true, that would be a major version bump rather than a minor one ('coz we figured out ways to have it with milk). So definitely not part of this PR.

Also, jumping from topic to topic is exactly the point. There are certain things we take to be true and about these we form entire worlds of understanding. Birds fly; gravity falls; stevens universe; Cinnamon Toast Crunch is good.